inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/KnowledgeBaseDetailsPanel.java

```diff
@@ -389,6 +390,16 @@ protected void setUpCommonComponents(WebMarkupContainer wmc) {
 
                 private static final long serialVersionUID = -2101263555896964046L;
 
+                @Override
+                protected void onConfigure()
+                {
+                    setEnabled(false);
+                }
+            });
+            wmc.add(new CheckBox("supportConceptLinking"){
+
+                private static final long serialVersionUID = -2101263555896964046L;
+
```

Comment:
Why is the `supportConceptLinking` always disabled? It should be enabled under the same conditions as used in the wizard - AFAIR it should be modifiable if the KB is remote.

Reply:
There are two modes - ViewMode and EditMode. In ViewMode it should not be possible to check the box.

Reply:
Shouldn't the it then be enabled based on the mode? Like `setEnabled(mode == EditMode)` or something like that?